.github/workflows/docker-image.yml

```diff
@@ -39,6 +40,9 @@ jobs:
           elif [ "${{ github.event.inputs.image }}" == "gravitino-ci-trino" ]; then
             echo "image_type=trino" >> $GITHUB_ENV
             echo "image_name=datastrato/gravitino-ci-trino" >> $GITHUB_ENV
+          elif [ "${{ github.event.inputs.image }}" == "gravitino-ci-doris" ]; then
```

Comment:
I'm afraid that you need to consider the architecture here, Doris does not support `all`
<img width="1559" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/e252cc09-c463-427b-8cf3-d4c6916e8572">
<img width="1370" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/2f26a373-3ed9-47a9-8364-2b80e05d4cab">

Reply:
Why doesn't support all?

Reply:
Doris should support all type.

Reply:
@zhoukangcn Could you raise a follow-up pr to solve this issue?

Reply:
Doris has releasing packages for both ARM and x86. You need to separately build the docker image for x86 and ARM.

Reply:
> Doris has different packages for ARM and x86 architectures. You need to separately build the docker image for x86 and ARM.

You can download them to the local machine at the same time. Copy the specific the package in the Docker image.

Reply:
The package of Doris is too large, with a single core file of BE having 3.8GB. To control the size of the Docker Image, it is recommended to build for ARM and X86 separately.

Reply:
We just download them, we don't copy all of them to the Docker image. we just copy the one which we need.

Reply:
I guess you mean, we can update `build-docker.sh`, we need build x86 and arm as two step in a single execution when executue with args `--platform=all` and `--type=doris` ?

Reply:
Now, Doris Image can build as follow：
```
./build-docker.sh --platform linux/arm64 --type doris --image doris_ci --tag test
./build-docker.sh --platform linux/amd64 --type doris --image doris_ci --tag test
```
can you do this in github workflows?